### PR TITLE
[SCB-2199] Add GitHub Actions workflows

### PR DESCRIPTION
**Pull request by coolbeevip**
* Add a CI workflow for PRs and push branch master

### CODE REVIEW

**Comment by zhfeng** [comment]
Thanks @coolbeevip - Nice work !

Is it possible to remove the travis configration if we make sure github flow works ?

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            !~/.m2/repository/org/apache/servicecomb
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+      - name: Show the first log message
+        run: git log -n1
+      - name: Build and Test and Install Docker
+        if: ${{ success() }}
+        run: ./mvnw clean install -B -Pdocker
+      - name: Upload unit test results when the previous step of a job fails
+        if: ${{ failure() }}
```
Comment (WillemJiang) [inline .github/workflows/ci.yaml:36]: Why we do this if the tests are failed?

**Comment by coolbeevip** [comment]
> Thanks @coolbeevip - Nice work !
> 
> Is it possible to remove the travis configration if we make sure github flow works ?

Build and tests passed in my forked repository  https://github.com/coolbeevip/servicecomb-pack/actions/runs/612410941 
But Sonar and Coverage have not been added yet. I will add sonar and coverage in other PRs and remove the travis config

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (coolbeevip) [reply .github/workflows/ci.yaml:36]: After the test fails, the unit test report will be uploaded to Github for easy download through the Github web interface

Comment (WillemJiang) [reply .github/workflows/ci.yaml:36]: It makes sense.

**Review by WillemJiang — APPROVED** [review]

**Comment by zhfeng** [comment]
@coolbeevip can you squash all the commits before merging ?

**Review by zhfeng — APPROVED** [review]

**Comment by coolbeevip** [comment]
> @coolbeevip can you squash all the commits before merging ?

done

**Pull request merged**